Eshop.Persistence/EshopDbContext.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Eshop.Domain;
+using Microsoft.EntityFrameworkCore;
+
+namespace Eshop.Persistence
+{
+    public class EshopDbContext : DbContext
+    {   
+        public EshopDbContext(DbContextOptions<EshopDbContext> options) : base(options) 
+        { 
+        
+        }
+
+        public DbSet<Category>? Categories  { get; set; }
```

Comment:
private set;
you need to pay attention on encapsulation (OOP the most critical rule that people ruin)
here you can see that is difference between understanding OOP principles in theory and implementation

Reply:
fixed, okay I will pay more attention to encapsulation...